content/general/information-technology/information-technology.md

```diff
@@ -0,0 +1,27 @@
+---
+Title: 'Information Technology'
+Description: 'Information Technology is the use of digital technological tools to store and turn data into useful information'
+Codecademy Hub Page: 'https://www.codecademy.com/catalog/subject/cybersecurity'
+CatalogContent:
+  - 'introduction-to-information-technology'
+---
+
+<link rel="cannonical" href="https://www.codecademy.com/courses/introduction-to-it/articles/it-overview"/>
+
+**Information Technology**, also known as IT, is the use of digital technological tools to store and turn data into useful information. Since we live in a technology-driven world, IT plays a major role in our lives whether its fixing a cracked screen on a phone, doing a data transfer from an old computer to another one, or repairing a PC's hardware.
+
+## Roles
+
+These are the common roles in the IT industry:
+
+- Hardware Technicians: Installs, repairs, and maintains [computer hardware](https://www.codecademy.com/resources/docs/general/computer-hardware). May also be required to troubleshoot hardware and software issues, as well as provide training and support to users on how to use the devices effectively.
+- Network Engineers: Designs,plans, and implements network infrastructures ranging from local area networks(LANs), wide area networks(WANs), and intranets for businesses and organizations. Also analyzes network traffic and discuss suggestions how to increase it to fulfill the organization/business' needs.
+- Desktop Support Personnel: Typically provides tech support(e.g.,restoring and backing up data to mobile devices) to customers via computer or phone.
```

Comment:
```suggestion
- Desktop Support Personnel: Typically provides tech support (e.g. restoring and backing up data to mobile devices) to customers via computer or phone.
```
Can we include more than one example, they do more than restoring/backing up data.